docs/tutorial/typesfuns.rst

```diff
@@ -495,9 +495,19 @@ vector. For example:
     testVec : Vect 4 Int
     testVec = 3 :: 4 :: 5 :: 6 :: Nil
 
-    inVect : IsElem 5 testVec
+    inVect : IsElem 5 Main.testVec
     inVect = There (There Here)
 
+
+.. important:: Implicit Arguments and Scope
+
+    Within the type signature the typechecker will treat all variables
+    that start with an lowercase letter as an implicit variable. To
```

Comment:
The rule is actually "all variables that start with a lowercase letter and are not applied to something else" --- I think this is the right place for it.